pepper-apis/config/application.conf.ctmpl

```diff
@@ -71,10 +71,11 @@
 {{end}}{{end}}
     "elasticsearchUsername": "pepper_backend",
     "elasticsearchBatchSize": 100,
-    "dsmJwtSecret": "{{$dsm_conf.Data.portal.jwtDdpSecret}}",
     "dbDsmInstance": "{{$conf.Data.dbDsmInstance}}",
     "dbApiInstance": "{{$conf.Data.dbApiInstance}}",
     "dbHousekeepingInstance": "{{$conf.Data.dbHousekeepingInstance}}",
+    "dsmJwtSecret": "{{$dsm_conf.Data.portal.jwtDdpSecret}}",
+    "dsmJwtSigner": "org.broadinstitute.kdux",
```

Comment:
Note: I have added new config key so folks will need to re-render configs.

I pulled this value out of the code and into config because I think it's better to have it here instead of hardcoded in the code. I am assuming this is not a secret (otherwise why did we check it into source code?) so I didn't put it into vault.